src/test/java/Homework17.java

```diff
@@ -0,0 +1,71 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Collection;
+
+public class Homework17 extends BaseTest {
+
+
+    @Test
+    public void addSongToPlaylist() throws InterruptedException {
+
+        String newSongNotification = "Added 1 song into";
+
+
+        navigateToPage();
+        provideEmail("natalia.kim01@testpro.io");
+        providePassword("AxKrdBnS");
+        clickSubmit();
+        searchSong("Lament");
+        clickViewAllBtn();
+        clickFirstSong();
+        clickAddTo();
+        choosePlaylist();
+        Assert.assertTrue(getNotificationText().contains(newSongNotification));
+
+
+    }
+    public void searchSong(String name) throws InterruptedException {
+        WebElement searchField = driver.findElement(By.cssSelector("div#searchForm input[type= 'search']"));
+        searchField.sendKeys(name);
+        Thread.sleep(2000);
+
+    }
+
+    public void clickViewAllBtn() throws InterruptedException {
```

Comment:
// comment from hw team: An InterruptedException is thrown when a thread is interrupted while it's waiting, sleeping, or otherwise occupied. In other words, some code has called the interrupt() method on our thread. It's a checked exception, and many blocking operations in Java can throw it.
If you implemented Explicit waits -> you cant remove throws InterruptedException after each function.